### PR TITLE
37- Controller, service and router created

### LINKED ISSUE #17: Create Controller and Services for Venues

### PR DESCRIPTION
**Pull request by DanielMJSilva**
I created the following files for venue:

Controller: getAllVenues and getOneVenueById
Service: getAllVenues and getVenueById
Route: venueRouter

All Venues:
[Image](https://user-images.githubusercontent.com/70338659/262428257-cd136444-1e6f-4178-9a59-4b52b1262789.png)

Venue by Id:
[Image](https://user-images.githubusercontent.com/70338659/262434776-d6506239-eb2e-4bd4-b93e-bdc589acb6c2.png)

### CODE REVIEW

File: src/services/venueService.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import prisma from '../config/prisma/prisma-client'
+
+
+export async function getAllVenues(): Promise<Venue[]> {
+  return prisma.venue.findMany()
+}
+
+
+export async function getVenueById(venueId: string) {
+    return prisma.venue.findUnique({
```
Comment (rmc3408) [inline src/services/venueService.ts:11]: Just add TypeScript here on return of function.. promise<venue>

File: src/services/venueService.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { type Venue } from '@prisma/client'
+import prisma from '../config/prisma/prisma-client'
+
+
+export async function getAllVenues(): Promise<Venue[]> {
+  return prisma.venue.findMany()
```
Comment (rmc3408) [inline src/services/venueService.ts:6]: Follow same as users, please Include activity.

File: src/config/prisma/prisma-client.ts [hunk 3]
```diff
@@ -46,8 +46,8 @@ export async function startORM() {
         },
       })
 
-      await prisma.$disconnect()
-      shutDown('database successful seeded')
+      // await prisma.$disconnect()
```
Comment (rmc3408) [inline src/config/prisma/prisma-client.ts:49]: Keep this on. It is to shutdown server.

**Review by rmc3408 — APPROVED** [review]
Tiny adjustment. Perfect

File: src/services/venueService.ts (hunk 2 above)
Comment (rmc3408) [reply src/services/venueService.ts:6]: On user services.. I added a query which has include activity: true

File: src/services/venueService.ts (hunk 1 above)
Comment (DanielMJSilva) [reply src/services/venueService.ts:11]: //TODO find return type for this function 

return is breaking with : Promise<Venue>

**Comment by DanielMJSilva** [comment]
TODO find type for this function

```
export async function getVenueById(venueId: string) {
    return prisma.venue.findUnique({
        where:{
            id: venueId
        }})
  
}
```

**Pull request merged**